tools/flecstan/README.rst

```diff
@@ -0,0 +1,53 @@
+.. |br| raw:: html
```

Comment:
I don't care too much, but why are we using `rst` instead of markdown (like all the docs in `developer/`?

Reply:
Perhaps I'm wrong, but I thought I understood from a conversation with @tuxfan that we were moving to .rst as in flecsi/sphinx/.

Reply:
Good point!